project_task_pull_request_state/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Cetmix <https://cetmix.com/>
```

Comment:
Please complete this section according to the latest updates in the DevMemo

```
Cetmix <https://cetmix.com/>
    Ivan Sokolov
    Vladimir Kalmykov
```

Reply:
Applied

Reply:
I'd suggest going with 
```
* `Cetmix <https://cetmix.com/>`__

  * Ivan Sokolov
  * Vladimir Kalmykov

```
Formatting, note the extra newline